src/Microsoft.DotNet.Interactive.CSharpProject/Packaging/BlazorPackage.cs

```diff
@@ -4,45 +4,44 @@
 using System;
 using System.IO;
 
-namespace Microsoft.DotNet.Interactive.CSharpProject.Packaging
+namespace Microsoft.DotNet.Interactive.CSharpProject.Packaging;
+
+public class BlazorPackage : PackageBase

```

Comment:
We should review this package architecture, the blazor package I am not even sure is needed the way it is anymore